app/src/main/res/layout/audio_fragment.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
```

Comment:
Are these data variables being used for anything? In general, I think we should also only add values to the view model if they're going to be used. Even if it's a simple text view, we should bind values to the UI that need to exist for the long term as placeholder bindings.

Reply:
Changed this file which uses data variables